packages/expandable-card/src/ExpandableCard.tsx

```diff
@@ -268,6 +267,8 @@ const ExpandableCard = ({
             <Icon
               className={cx(iconStyle, iconTransitionStyle[state])}
               glyph="ChevronUp"
+              size={24}
+              fill={palette.gray.base}
```

Comment:
I think we should hardcode this value instead of using `palette` since we're moving away from using `palette`.